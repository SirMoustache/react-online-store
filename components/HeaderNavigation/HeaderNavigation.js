/**
 * Absolute imports
 */
import React from 'react';
import styled from 'styled-components';
import Link from 'next/link';

/**
 * Global Components
 */
import A from '../A';

const Root = styled.div`
  flex: 0 0 auto;
`;

const Header = () => (
  <Root>
    <Link href="/">
      <A>Home</A>
    </Link>
    <Link href="/about">
      <A>About</A>
    </Link>
  </Root>
);

export default Header;
