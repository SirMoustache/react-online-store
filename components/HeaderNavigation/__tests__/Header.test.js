/**
 * Absolute imports
 */
import React from 'react';
import { shallow } from 'enzyme';

/**
 * Local Components
 */
import HeaderNavigation from '..';

describe('<HeaderNavigation />', () => {
  it('should match snapshot', () => {
    const renderedComponent = shallow(<HeaderNavigation />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
