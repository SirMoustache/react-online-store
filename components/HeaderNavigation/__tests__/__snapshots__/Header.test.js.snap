// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderNavigation /> should match snapshot 1`] = `
<div>
  <nav
    class="HeaderNavigation__Root-ixstj1-1 buTUPC"
  >
    <a
      class="HeaderNavigation__NavLink-ixstj1-0 bzKWdB A-sc-1g6vast-0 YkdzD"
    >
      Home
    </a>
    <a
      class="HeaderNavigation__NavLink-ixstj1-0 bzKWdB A-sc-1g6vast-0 YkdzD"
    >
      About
    </a>
    <a
      class="HeaderNavigation__NavLink-ixstj1-0 bzKWdB A-sc-1g6vast-0 YkdzD"
    >
      Shop
    </a>
  </nav>
</div>
`;
